Stripe/StripeiOSTests/STPPaymentHandlerStubbedMockedFilesTests.swift

```diff
@@ -0,0 +1,561 @@
+                                 formSpecPaymentHandler: paymentSheetFormSpecHandler)
+    }
+
+    private func replaceData(data: Data, variables: [String: String]) -> Data {
+        var template = String(data: data, encoding: .utf8)!
+        for (templateKey, templateValue) in variables {
+            let translated = template.replacingOccurrences(of: templateKey, with: templateValue)
+            template = translated
+        }
+        return template.data(using: .utf8)!
+    }
+
+    private func stubConfirm(fileMock: FileMock, responseCallback: ((Data) -> Data)? = nil) {
+        stub { urlRequest in
+            return urlRequest.url?.absoluteString.contains("/confirm") ?? false
+        } response: { urlRequest in
+            let mockResponseData = try! fileMock.data()
+            let data = responseCallback?(mockResponseData) ?? mockResponseData
+            return HTTPStubsResponse(data: data, statusCode: 200, headers: nil)
+        }
+    }
+    private func stubRetrievePaymentIntent(
+        fileMock: FileMock,
+        responseCallback: ((Data) -> Data)? = nil
+    ) {
+        stub { urlRequest in
+            return urlRequest.url?.absoluteString.contains("/payment_intents") ?? false
+        } response: { urlRequest in
+            let mockResponseData = try! fileMock.data()
+            let data = responseCallback?(mockResponseData) ?? mockResponseData
+            return HTTPStubsResponse(data: data, statusCode: 200, headers: nil)
+        }
+    }
+    private func affirmSpec(redirectStrategy: String) -> [NSDictionary] {
+        let formSpec =
+            """
+            [{
+                "type": "affirm",
+                "async": false,
+                "fields": [
+                    {
+                        "type": "affirm_header"
+                    }
+                ],
+                "next_action_spec": {
+                    "confirm_response_status_specs": {
+                        "requires_action": {
+                            "type": "redirect_to_url",
+                            "native_mobile_redirect_strategy": "\(redirectStrategy)"
+                        }
+                    },
+                    "post_confirm_handling_pi_status_specs": {
+                        "succeeded": {
+                            "type": "finished"
+                        }
+                    }
+                }
+            }]
+            """.data(using: .utf8)!
+        return try! JSONSerialization.jsonObject(with: formSpec) as! [NSDictionary]
+    }
+}
+extension STPPaymentHandlerStubbedMockedFilesTests {
+    func authenticationPresentingViewController() -> UIViewController {
+        return UIViewController()
+    }
+}
+
+public class ClassForBundle {}
+@_spi(STP) public enum FileMock: String, MockData {
+    public typealias ResponseType = StripeFile
+    public var bundle: Bundle { return Bundle(for: ClassForBundle.self) }
+
+    case paymentIntentResponse = "paymentIntentResponse"
```

Comment:
Note: I don't know why, but I had to change this from `MockFiles/paymentIntentResponse` in order for it to locate the file.

Reply:
probably cause i added the folder to the project last time. seems fine to me!